assets/wizards/popups/components/segment-group/index.js

```diff
@@ -78,25 +79,94 @@ const SegmentGroup = props => {
 	}
 	return (
 		<Card isSmall className="newspack-campaigns__segment-group__card">
-			<h3 className="newspack-campaigns__segment-group__card__segment">
-				<a href={ `#/segments/${ id }` }>
-					{ id ? __( 'Segment: ', 'newspack' ) : '' }
-					{ label }
+			<div className="newspack-campaigns__segment-group__card__segment">
+				<div className="newspack-campaigns__segment-group__card__segment-title">
+					<h3>
+						{ id ? (
+							<Button
+								href={ `#/segments/${ id }` }
+								label={ __( 'Edit Segment ', 'newspack' ) }
+								isLink
+								showTooltip
+								tooltipPosition="bottom center"
+							>
+								{ __( 'Segment: ', 'newspack' ) }
+								{ label }
+							</Button>
+						) : (
+							label
+						) }
+					</h3>
 					<span className="newspack-campaigns__segment-group__description">
-						{ descriptionForSegment( segment, categories ) }
+						{ id ? descriptionForSegment( segment, categories ) : __( 'No segment', 'newspack' ) }
```

Comment:
Suggestion: "All readers regardless of segment"

Reply:
Good idea, updated in ccc3f17c.